server/api/lineitems.js

```diff
@@ -1,8 +1,23 @@
 const router = require('express').Router();
+const Product = require('../db/Product');
+const Order = require('../db/Order');
 module.exports = router;
 
-router.get('/', (req, res) => {
+router.get('/', (req, res, next) => {
+  Order.findById(req.params.orderId)
+    .then(order => res.send(order))
+})
+
+router.post('/', (req, res, next) => {
+  const { userId, productId } = req.body;
+
+ Order.addLineItem({ userId, productId })
+   .then(order => res.send(order))
+   .catch(next);
 });
 
-router.delete('/', (req, res) => {
+router.delete('/:lineItemId', (req, res, next) => {
```

Comment:
Same logic here, if we use Order model, then let's keep it consistent, and put it in the orders.js